platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/workflow/export/WorkflowDtoV1.java

```diff
@@ -0,0 +1,70 @@
+package com.softicar.platform.workflow.module.workflow.export;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import java.util.List;
+
+public class WorkflowDtoV1 {
```

Comment:
I think we should add a constant to this class, e.g. `public static final int workflowApiVersion = 1`, in order to have this constant in the exported JSON.

Rationale: In a future in which `WorkflowDtoV2`, `WorkflowDtoV3`, etc. already exist, compatibility upon import will be a concern. Recornizing a non-current DTO version would be as easy as reading the value from `workflowApiVersion`. We could then either bluntly reject a JSON file with a non-current version, or apply some conversion logic, or use a different importer, as appropriate.